lib/index.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     'block-spacing': require('./rules/block-spacing'),
     'brace-style': require('./rules/brace-style'),
     'camelcase': require('./rules/camelcase'),
+    'class-order': require('./rules/class-order'),
```

Comment:
I think the name of the rule is misleading.
People may misunderstand that it targets `:class` or JS `class`.

Could you change the rule name?

For example, what do you think about changing the name of the rule to "static-class-names-order"?

Reply:
done